pkg/trace/api/otlp.go

```diff
@@ -499,3 +539,73 @@ func spanKindName(k pdata.SpanKind) string {
 	}
 	return name
 }
+
+// conventionsMappings defines the mapping between OpenTelemetry semantic conventions
+// and Datadog Agent conventions
+var conventionsMapping = map[string]string{
```

Comment:
I assume this is a copy of https://github.com/DataDog/datadog-agent/blob/62f28598d2cbe8dccf5a9c56260fbc264b256d33/pkg/otlp/model/attributes/attributes.go#L28 ? If so, let's try and use that (you could use `TagsFromAttributes` or if that is not good for some reason, create a new public function on `pkg/otlp/model/attributes`)

Reply:
Not sure what you mean here. Do you mean to import `datadogexporter` ?